deploifai/project/create.py

```diff
@@ -2,17 +2,129 @@
+                "message": "Choose a workspace",
+                "choices": [
+                    {"name": ws["username"], "value": ws} for ws in workspaces_from_api
+                ],
+            }
+        )
+        if _choose_workspace == {}:
+            raise click.Abort()
+        command_workspace = _choose_workspace["workspace"]
+
+    # ensure project name is unique to the chosen workspace
+    try:
+        project_names_dict = deploifai_api.get_project_names(workspace=command_workspace)
+    except DeploifaiAPIError as err:
+        click.echo("An error occured when fetching projects. Please try again.")
+        return
+
+    err_msg = "Project name taken. Choose a unique project name:"
+
+    project_names = []
+
+    for project_name_dict in project_names_dict:
+        project_names.append(project_name_dict["name"])
+
+    name_taken_err_msg = f"Project name taken. Existing names in chosen workspace: {' '.join(project_names)}\nChoose a unique project name:"
+    err_msg = name_taken_err_msg
+
+    is_valid_name = not(name in project_names)
+
+    while not is_valid_name:
+        prompt_name = prompt(
+            {
+                "type": "input",
+                "name": "project_name",
+                "message": err_msg,
+            }
+        )
+
+        new_project_name = prompt_name["project_name"]
+
+        if len(new_project_name) == 0:
+            err_msg = "Project name cannot be empty.\nChoose a non-empty project name:"
+        elif not new_project_name.isalnum():
+            err_msg = f"Project name should only contain alphanumeric characters.\nChoose a valid project name:"
+        elif new_project_name in project_names:
+            err_msg = name_taken_err_msg
+        else:
+            name = new_project_name
+            is_valid_name = True
+
+    try:
+        cloud_profiles = deploifai_api.get_cloud_profiles(
+            workspace=command_workspace
+        )
+    except DeploifaiAPIError as err:
+        click.echo("Could not fetch cloud profiles. Please try again.")
+        return
+
+    # TODO: prompt user if no existing cloud profiles exist
+    choose_cloud_profile = prompt(
+        {
+            "type": "list",
+            "name": "cloud_profile",
+            "message": "Choose a cloud profile for project",
+            "choices": [
+                {
+                    "name": "{name}({workspace}) - {provider}".format(
+                        name=cloud_profile.name,
+                        workspace=cloud_profile.workspace,
+                        provider=cloud_profile.provider,
+                    ),
+                    "value": cloud_profile,
+                }
+                for cloud_profile in cloud_profiles
+            ],
+        }
+    )
+
+    cloud_profile = choose_cloud_profile["cloud_profile"]
 
-    local_config.set_project_config(name, context.local_config)
+    project_id = deploifai_api.create_project(name, cloud_profile)
```

Comment:
I'm pretty sure `deploifai_api.create_project` returns a dictionary with the key `id`. Not the `id` string itself.